packages/ui/src/components/molecules/RightIconButton/RightIconButton.tsx

```diff
@@ -34,19 +37,28 @@ const RightIconButton = ({
   onPress = () => {},
   title = null,
   icon,
+  placeholder,
 }: RightIconProps) => {
+  const Colors = useThemeColor();
+
+  const displayPlaceholder = useMemo(() => {
+    return checkNullString(title) && !checkNullString(placeholder);
+  }, [placeholder, title]);
+
   return (
     <TouchableOpacity onPress={onPress} activeOpacity={0.9}>
       <Card style={[styles.container, style]}>
-        <Text style={[styleText, styles.text]} numberOfLines={2}>
-          {title != null ? title : ''}
+        <Text
+          style={[styleText, styles.text]}
+          textColor={displayPlaceholder && Colors.placeholderTextColor}
+          numberOfLines={2}>
+          {!checkNullString(title) ? title : displayPlaceholder && placeholder}
```

Comment:
Do not modify this component. The Picker is not its main purpose and everything you are doing is doable from outside.